src/tests/functional/plugin/shared/include/behavior/executable_network/exec_network_base.hpp

```diff
@@ -70,13 +70,13 @@ TEST_P(ExecutableNetworkBaseTest, checkGetMetric) {
 
 TEST_P(ExecutableNetworkBaseTest, canLoadCorrectNetworkToGetExecutableAndCheckConfig) {
     auto execNet = ie->LoadNetwork(cnnNet, target_device, configuration);
-    if (target_device == CommonTestUtils::DEVICE_AUTO) {
-        // AUTO executable network didn't support to read any config.
-        GTEST_SKIP();
-    }
     for (const auto& configItem : configuration) {
         InferenceEngine::Parameter param;
-        ASSERT_NO_THROW(param = execNet.GetConfig(configItem.first));
+        try {
```

Comment:
Why do you remove `ASSERT_NO_THROW`? I suppose we should do not change

Reply:
If AUTO/MULTI can't support this test case, skip it in skip file and with a JIRA ticket to follow up.